### PR TITLE
Draft of three apimon roles - epmon, scheduler, executor

### PR DESCRIPTION
**Pull request by kucerakk**

### CODE REVIEW

File: roles/scheduler/templates/env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+APIMON_IMAGE={{ executor_image }}
```
Comment (vladimirhasko) [inline roles/scheduler/templates/env]: not needed

File: roles/epmon/templates/env [hunk 2]
```diff
@@ -0,0 +1,2 @@
+APIMON_IMAGE={{ epmon_image }}
+APIMON_SCHEDULER_IMAGE={{ scheduler_image }}
```
Comment (vladimirhasko) [inline roles/epmon/templates/env]: not needed

File: roles/executor/templates/env [hunk 3]
```diff
@@ -0,0 +1,2 @@
+APIMON_IMAGE={{ executor_image }}
+APIMON_SCHEDULER_IMAGE={{ scheduler_image }}
```
Comment (vladimirhasko) [inline roles/executor/templates/env]: not needed

File: roles/scheduler/templates/env (hunk 1 above)
Comment (kucerakk) [reply roles/scheduler/templates/env]: ok, separated the env file per role, this way each role has its own env file and its own image variable

File: roles/epmon/templates/env (hunk 2 above)
Comment (kucerakk) [reply roles/epmon/templates/env]: ok, separated the env file per role, this way each role has its own env file and its own image variable

File: roles/executor/templates/env (hunk 3 above)
Comment (kucerakk) [reply roles/executor/templates/env]: ok, separated the env file per role, this way each role has its own env file and its own image variable

**Review by gtema — DISMISSED** [review]

**Review by vladimirhasko — DISMISSED** [review]

**Comment by kucerakk** [comment]
regate

**Comment by kucerakk** [comment]
recheck

**Comment by gtema** [comment]
recheck

**Comment by gtema** [comment]
recheck

**Comment by gtema** [comment]
recheck

**Comment by kucerakk** [comment]
regate

**Review by vladimirhasko — APPROVED** [review]

**Pull request merged**